packages/data-context/src/data/ProjectConfigManager.ts

```diff
@@ -461,6 +461,15 @@ export class ProjectConfigManager {
     )
   }
 
+  private getRepoRoot = () => {
```

Comment:
```suggestion
  get getRepoRoot () {
```

Can be a getter, no side effects here

Reply:
I switched from `private` to `get` and then it was failing as it no longer was a property on `this` 🥲 I'm not currently familiar with using get/private in JavaScript, but it seems fine to leave as `private` since at this time it is only intended to be used in this file